po/zh-CN.po

```diff
@@ -9153,13 +9153,13 @@ msgstr ""
 
 #: src/traits/trait-objects.md:1
 msgid "# Trait Objects"
-msgstr ""
+msgstr "# 特征对象"
```

Comment:
```suggestion
msgstr "# 特征（Trait）对象"
```
Keywords in the language or specific concepts are often not translated, such as in https://rustwiki.org/zh-CN/book/ch10-02-traits.html.

Reply:
Nice, I would do the same for a Danish translation :slightly_smiling_face: